.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,29 @@
+name: Lint on pull request
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  full_ci:
+    strategy:
+      matrix:
+        go-version: [1.14.x]
+    runs-on: ubuntu-18.04
+    steps:
+      # Get the code into the container
+      - name: Checkout
+        uses: actions/checkout@v2
+      # Test the code
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v2.5.2
+      - name: Test code
```

Comment:
did this step work without setting the test environment vars from github secrets?

https://docs.github.com/en/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow

Reply:
It did not but should now!